### ISSUE TITLE
Example blog post on ropensci.org

### ISSUE THREAD

**Issue by steffilazerte** [ropensci-org/blog-guidance#232]
Going through some of the technical stuff, it occurred to me that it might be nice to have a blog post on the real website demonstrating what many of these options look like. 

This would be a nice reference to use to see

1. If any website changes broke any of the options
2. What options are available
3. What the differences among specific variants look like (e.g., the many ways of creating block quotes)

The only thing would be if this post would see weird on the website... a couple of options

1. Just explain it's purpose in the intro paragraph, linking to the blog guide (no bad thing!)
2. Try to hide it from the blog list and search? Just available through a link on the Blog Guide.

@maelle, @yabellini, thoughts?

**Comment by maelle** [comment]
I like this idea, I'll try to work on it soon. The idea would be to add a possible YAML field called "unlisted" and to amend all themes files (for the blog list, archive list, author list, normal RSS feed, Rogue RSS, R-Bloggers RSS, etc.) so that they don't mention the page.

**Comment by maelle** [comment]
Or, to make things easier and faster, why not create this post in a PR and use Netlify PR preview link?

Also the occasion to demo the PR review. 

I won't work on this before I know what strategy you find better @steffilazerte @yabellini (and for the second strategy my help is not needed I think).

**Comment by maelle** [comment]
If it's a PR we'd need to rebase it often.

**Comment by steffilazerte** [comment]
I think my preference is to have it be a 'live' post and not live in a PR. Just seems neater?

**Comment by maelle** [comment]
I've thought about it some more and I'm really hesitant to make edits to many theme files for just one post. 

I've just thought about adding it as a draft post or a post very far into the future (like, in 3024) since [by default Hugo does not build those](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content). Deploying it would not be possible easily (as we'd also need to make sure the pages can't be indexed by Google) but we could tell people to build it locally using `hugodown::hugo_build()` (that exposes the Hugo option to build future/draft post) and to serve it using `servr::httw()`. Con: no live preview.

Yet another alternative: you write an actual post presenting the blog guidance and how it's been helpful to us, and as a constraint you use all the formatting tricks :joy: 

To summarize

1. Making it possible to have one post up but not indexed from anywhere including not from Google. I am currently really not in favor of this as it implies at lot of theme edits and I'd also be worried to miss one file.
2. Example post as a live PR. Rebase the PR regularly. Webmaster looks at it when making changes to the theme.
3. Example post as a draft/future/expired post. Authors build it locally. Webmaster looks at it when making changes to the theme.
4. Example post disguised as a post about the blog guidance. Webmaster looks at it when making changes to the theme.

**Comment by yabellini** [comment]
I like the last option the most. It is the less disruptive one.

**Comment by steffilazerte** [comment]
I agree, I think we could even write the post and be explicit that this is also a post meant to demo all the tricks so that future blog writers can see how they work in action. No need for a disguise, really 😁

**Comment by maelle** [comment]
Awesome! Then nothing for me to do... except fix the bugs you might/will discover :grin:
